io.github.dfandrich.gpscorrelate.json

```diff
@@ -26,44 +39,45 @@
           "cleanup": [
               "/bin",
               "/include",
-              "/lib/cmake",
               "/lib/pkgconfig",
               "/lib/*.a",
-              "/share/man"
+              "/share/cmake/exiv2/*",
+              "/share/man/man1/*"
           ],
           "sources": [
               {
                   "type": "archive",
-                  "url": "https://github.com/Exiv2/exiv2/releases/download/v0.27.6/exiv2-0.27.6-Source.tar.gz",
-                  "sha256": "4c192483a1125dc59a3d70b30d30d32edace9e14adf52802d2f853abf72db8a6"
+                  "url": "https://github.com/Exiv2/exiv2/archive/refs/tags/v0.28.2.tar.gz",
+                  "sha256": "543bead934135f20f438e0b6d8858c55c5fcb7ff80f5d1d55489965f1aad58b9"
               }
           ]
         },
         {
             "name": "gpscorrelate-gui",
             "buildsystem": "simple",
             "build-commands": [
-                /* gettext catalogues are dropped if stored in the standard location, so move them */
-                "make -j${FLATPAK_BUILDER_N_JOBS} prefix=${FLATPAK_DEST} localedir=${FLATPAK_DEST}/share/gpscorrelate/locale CFLAGS=\"${CFLAGS} -DENABLE_NLS=1\"",
+                "make -j${FLATPAK_BUILDER_N_JOBS} prefix=${FLATPAK_DEST} localedir=${FLATPAK_DEST}/share/gpscorrelate/locale CFLAGS=\"${CFLAGS} -DENABLE_NLS=1\" CXXFLAGS=\"${CXXFLAGS} -DENABLE_NLS=1\"",
                 "make prefix=${FLATPAK_DEST} localedir=${FLATPAK_DEST}/share/gpscorrelate/locale install install-po",
+                "sed -i 's/Icon=gpscorrelate-gui/Icon=io.github.dfandrich.gpscorrelate/' gpscorrelate.desktop",
                 "cp gpscorrelate.desktop ${FLATPAK_ID}.desktop",
                 "desktop-file-install --vendor= --set-icon=${FLATPAK_ID} --dir=/app/share/applications ${FLATPAK_ID}.desktop",
                 "install -m0644 -Dp gpscorrelate-gui.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg",
+                "install -m0644 -Dp gpscorrelate-gui.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/gpscorrelate-gui.svg",
                 "install -m0644 -Dp ${FLATPAK_ID}.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo",
-                "make check"
+                "make check",
+                "install -m0755 -d ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps",
+                "gdk-pixbuf-thumbnailer -s 48 gpscorrelate-gui.svg ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png",
+                "gdk-pixbuf-thumbnailer -s 48 gpscorrelate-gui.svg ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/gpscorrelate-gui.png"
```

Comment:
you shouldn't need to do so since there is a SVG.

Reply:
This was a hack to figure out why the build is failing. Adding png icons didn't actually seem to make any difference.

Reply:
See the manpage for `flatpak-manifest` (or the online doc)